### PR TITLE
archive some old docs and add some to the sidebar

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
During scoping out work to ensure all live docs are in our docs sidebar I did some quick fixes - archiving some old content and adding some docs to the sidebar where there is an obvious spot to put them.

This PR archives 10 pages of outdated content, adds some redirects for some where relevant and adds some pages into the sidebar that were missing.

# Reasons
https://circleci.atlassian.net/browse/DOCSTEAM-278?atlOrigin=eyJpIjoiOWFlZjQ4YmZmZjgxNGQxYThlZTlmNmY4ZTdlNTJiMmYiLCJwIjoiaiJ9

### CODE REVIEW

File: jekyll/_cci2/orb-author-validate-publish.md [hunk 1]
```diff
@@ -8,12 +8,16 @@ version:
 
 This guide covers the steps required to create a simple orb, manually, without using the orb development kit. We recommend the orb development kit for most orb projects, to find out more, see the [Orb Authoring Guide]({{site.baseurl}}/2.0/orb-author).
 
+## Create a namespace
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-author-validate-publish.md:11]:
```suggestion
## Create a namespace
{: #create-a-namespace }
```

File: jekyll/_cci2/orb-author-validate-publish.md [hunk 2]
```diff
@@ -8,12 +8,16 @@ version:
...
+
 1. If you have not already done so, claim a namespace for your user/organization using the following command, substituting your namespace choice and GitHub organization name:
 ```shell
 circleci namespace create <my-namespace> github <my-gh-org>
 ```
 **Note:** When creating a namespace via the CircleCI CLI, be sure to specify the VCS provider.
 
+## Create your orb
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-author-validate-publish.md:19]:
```suggestion
## Create your orb
{: #create-your-orb }
```

File: jekyll/_cci2/orb-author-validate-publish.md [hunk 3]
```diff
@@ -39,11 +43,15 @@ commands:
             - run: echo "Hello, << parameters.to >>"
 ```
 
+## Validate your orb
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-author-validate-publish.md:46]:
```suggestion
## Validate your orb
{: #validate-your-orb }
```

File: jekyll/_cci2/orb-author-validate-publish.md [hunk 4]
```diff
@@ -39,11 +43,15 @@ commands:
...
+
 1. Validate your orb code using the CLI:
 ```
 circleci orb validate /tmp/orb.yml
 ```
 
+## Publish your orb
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-author-validate-publish.md:53]:
```suggestion
## Publish your orb
{: #publish-your-orb }
```

File: jekyll/_cci2/orb-author-validate-publish.md [hunk 5]
```diff
@@ -59,6 +67,8 @@ circleci orb publish promote <my-namespace>/<my-orb-name>@dev:first patch
 circleci orb source <my-namespace>/<my-orb-name>@0.0.1
 ```
 
+## List available orbs
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-author-validate-publish.md:70]:
```suggestion
## List available orbs
{: #list-available-orbs }
```

File: jekyll/_cci2/orb-author-validate-publish.md [hunk 6]
```diff
@@ -71,4 +81,6 @@ To list **[private]({{site.baseurl}}/2.0/orb-intro/#private-orbs)** orbs:
 circleci orb list <my-namespace> --private
 ```
 
+## Next Steps
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-author-validate-publish.md:84]:
```suggestion
## Next Steps
{: #next-steps }
```

**Comment by rosieyohannan** [comment]
@Mayuko-Sa I've had to update a link in a Japanese page to get this build to pass because there was a link to an english page that I have archived.

**Review by Mayuko-Sa — APPROVED** [review]

**Pull request merged**